### PR TITLE
Fix message when there are no celebrations

### PR DESCRIPTION
**Pull request by inigomarquinez**
### Main changes

- :bug: amend published message when there are no celebrations

### Other changes

- 🔄 split publishers in different files for better readability
- 🔄 split slack message builder for celebrations for better readability
- :memo: improve README

### Screenshots

Captura de pantalla 2023-03-17 a las 9 44 43

### CODE REVIEW

**Comment by inigomarquinez** [comment]
@all-contributors please add @totegsito for bugs, code, doc, maintenance, review

**Comment by inigomarquinez** [comment]
@all-contributors please add @betisman for review

**Comment by inigomarquinez** [comment]
Adds @totegsito as a contributor for bug, code, doc, maintenance, review.

**Comment by inigomarquinez** [comment]
@all-contributors please add @totegsito for bugs, code, doc, maintenance, review

**Comment by inigomarquinez** [comment]
@all-contributors please add @totegsito for bugs, code, doc, maintenance, review

**Comment by inigomarquinez** [comment]
@all-contributors please add @betisman for review

**Comment by inigomarquinez** [comment]
@all-contributors please add @w3dani for infra, review

File: src/publishers/bank-holidays-publisher.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          text: `*${holiday.name}*\n${moment(holiday.start).format(
+            HUMAN_READABLE_DATE
+          )}`,
+        },
+      }))
+    );
+
+    monthBankHolidaysBlocks.push({
+      type: 'divider',
+    });
```
Comment (Betisman) [inline src/publishers/bank-holidays-publisher.ts:48]: There's no need of this, please feel free to ignore it.
```suggestion
    const bankHolidaysHeader = {
      type: 'context',
      elements: [
        {
          type: 'mrkdwn',
          text: '*This month bank holidays:*',
        },
      ],
    };
    
    const holiday2Block = holiday => ({
        type: 'section',
        text: {
          type: 'mrkdwn',
          text: `*${holiday.name}*\n${moment(holiday.start).format(
            HUMAN_READABLE_DATE
          )}`,
        },
      });
      
    const bankHolidaysData = bankHolidays.map(holiday2Block);
    
    const dividerBlock = {
      type: 'divider',
    };

    monthBankHolidaysBlocks = [
      ...monthBankHolidaysBlocks,
      bankHolidaysHeader,
      ...bankHolidaysData,
      dividerBlock,
      ];
```

**Review by Betisman — APPROVED** [review]
Nice job.

Next time we could split code changes and doc changes into different commits.

**Review by totegsito — APPROVED** [review]
👏 🫠

File: README.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ After successful deployment, you can invoke the deployed function by using the f
 serverless invoke --function main
 ```
 
-## Local development
+## 🏗️ Local development
```
Comment (totegsito) [inline README.md:61]: Sorry @inigomarquinez my comment was misplaced. The guide I was asking about is related to this environment variables. Where can all these variables be retrieved from?

Comment (inigomarquinez) [reply README.md:61]: @totegsito , I've referenced this in a new issue to improve the documentation: https://github.com/guidesmiths/bamboohrbot/issues/29

**Pull request merged**